scene/main/node.cpp

```diff
@@ -409,9 +409,7 @@ void Node::set_physics_process_internal(bool p_process_internal) {
 	if (data.physics_process_internal == p_process_internal)
 		return;
 
-	data.physics_process_internal = p_process_internal;
```

Comment:
if you remove this, then it wont be possible to query whether the node is doing such processing

Reply:
Is this not set just below this in line 419/417 as well?

Reply:
oh, that makes sense, didnt see as far head. Seems to be repeated in all the set process calls